internal/analyzers/skippers/skippers.go

```diff
@@ -69,6 +76,16 @@ func (sm *skipper) ShouldSkip(data collectors.CollectedData, violation opa_engin
 	return false
 }
 
+func (sm *skipper) ignoredPolicy(policy opa_engine.QueryResult) bool {
+	for _, ignored := range sm.ignoredPolicies {
+		if strings.Contains(policy.FullyQualifiedPolicyName, ignored) {
```

Comment:
Why use string contains and not equal? what will happen when we have one policy which is the prefix of an ignored policy?